apps/mobile/src/screens/PostScreen/MiniPostOpenGraphPreview.tsx

```diff
@@ -0,0 +1,116 @@
+import clsx from 'clsx';
+import { ResizeMode } from 'expo-av';
+import { Text, View } from 'react-native';
+import FastImage from 'react-native-fast-image';
+import CloseBracket from 'src/icons/CloseBracket';
+import OpenBracket from 'src/icons/OpenBracket';
+import { noop } from 'swr/_internal';
+
+import { RawProfilePicture } from '~/components/ProfilePicture/RawProfilePicture';
+import { Typography } from '~/components/Typography';
+
+type Props = {
+  username: string;
+  caption?: string;
+  imageUrl: string;
+  profileImageUrl?: string;
+};
+
+export default function MiniPostOpenGraphPreview({
+  username,
+  caption,
+  imageUrl,
+  profileImageUrl,
+}: Props) {
+  const letter = username?.[0]?.toUpperCase();
+  const fallbackProfilePicture = (
+    <RawProfilePicture
+      eventElementId="ProfilePicture"
+      eventName="ProfilePicture pressed"
+      eventContext={null}
+      size="xxs"
+      letter={letter ?? '?'}
+    />
+  );
+
+  return (
+    <View className="flex w-[358]px h-[188px] bg-[#F9F9F9] items-center justify-center">
+      <View className="flex flex-row items-center space-x-5">
+        <OpenBracket />
+        <View className="flex flex-row space-x-4">
+          <ImagePreview tokenUrl={imageUrl} onError={noop} />
+          <View className="flex flex-col justify-center">
+            <View className="flex flex-row items-center space-x-1">
+              {profileImageUrl ? (
+                <ImagePreview isPfp tokenUrl={profileImageUrl ?? ''} onError={noop} />
```

Comment:
not sure how to error handle with `onError` for the ImagePreview here and on L41

Reply:
Maybe can mark it as failed, just like in `NftPreviewAsset.tsx`

```tsx
  const handleError = useCallback(() => {
    markTokenAsFailed(
      token.dbid,
      new CouldNotRenderNftError('RawNftPreviewAsset', 'Failed to render', { id: token.dbid })
    );
  }, [markTokenAsFailed, token.dbid]);
```